gigascience/gigadb-website
# Upgrade beta submission wizard layout for Study and Author pages

# Pull request for issue: #1080

NOTE: this work in on hold until work on the submission wizard project is resumed

This is a pull request for the following functionalities:

* Upgrade beta submissin page to new layout
* Upgrade form styles for Study and Author pages

Study
![image](https://github.com/gigascience/gigadb-website/assets/143437854/a319d804-2f99-4776-af98-e143a45914e7)
![image](https://github.com/gigascience/gigadb-website/assets/143437854/cb8d613e-311f-40d3-aead-40b4d1a2bf27)

Error
![image](https://github.com/gigascience/gigadb-website/assets/143437854/89251028-5864-424f-903a-891ab693f91d)

Author
![image](https://github.com/gigascience/gigadb-website/assets/143437854/cf6167fc-eae9-4392-b608-3f618cb1b2ee)


## How to test?

* Navigate to http://gigadb.gigasciencejournal.com/datasetSubmission/create1
* Review page appearance
* fill in form and click next
* review page appearance of auhor page (left as work in progress, see comment thread in #1080 )

## How have functionalities been implemented?

* Upgrade Study and Author pages to use new layout
* Upgrade Study form in terms of layout and accessibility
  * Use readonly inputs or aria-disabled instead of `disabled`
  * Make input descriptions always visible and associated to inputs

## Any issues with implementation?

This multi-step form is relatively complex and it looks like work in progress so it seems better to put its restyling / a11y improvement on hold until new confirmation

## Any changes to automated tests?

--
